build-package

```diff
@@ -7,6 +7,9 @@ STARTDIR="$(pwd)"
 DESTDIR="$STARTDIR/pkg"
 OUTDIR="$STARTDIR/deb"
 
+# Remove potential leftovers from a previous build
+rm -rf $DESTDIR
```

Comment:
`$DESTDIR` needs to be quoted: `rm -rf "$DESTDIR"`.

This is because otherwise if there's a space in the path it'll be interpreted as two arguments.